fastembed/embedding.py

```diff
@@ -114,7 +115,7 @@ def __init__(
 
         # Hacky support for multilingual model
         self.exclude_token_type_ids = False
-        if model_name == "intfloat/multilingual-e5-large":
+        if model_name in ["intfloat/multilingual-e5-large", "xenova/multilingual-e5-large", "xenova/paraphrase-multilingual-mpnet-base-v2"]:
```

Comment:
Can we replace this with a property in models.json? 

E.g. add languages? And anyone which has more than "en" goes has this if True? That adds more work though, since we don't always have a complete model card.

Or just check for "multilingual" in the model_name for now? That's better than remembering to add every multilingual model to a separate list. This makes accepting community contributions harder.

Reply:
https://github.com/qdrant/fastembed/pull/103/commits/565fbf35a6d4232d08b4de3943819b214443b9af